Create message.js file and modify index.html

sebagabs opened the pull request:
Create the `message.js` script file and add it in the `index.html` file. Somehow this branch got a commit of create `RandomStringServlet.java` file, but I had done that on another PR; don't know what happened there. 

These lines are place holders, I plan to modify them. As of right now, I don't understand a lot of what is going on, so I wanna read about a bit more.

sebagabs committed:
Create RandomStringServlet.java file

sebagabs committed:
Link new message.js file to index.html

sebagabs committed:
Modify index.html file to implement displayMessage() func from message.js

sebagabs committed:
Create message.js file

SamRod33 commented:
The only thing I can think of about the accidental commit to this branch with `RandomStringServelt.java` is if you were on this branch and pushed the file. Always do a `git status` before committing helps me know what I am controlling. Hope this helps!

GoogleyGabriel commented:
> The only thing I can think of about the accidental commit to this branch with `RandomStringServelt.java` is if you were on this branch and pushed the file. Always do a `git status` before committing helps me know what I am controlling. Hope this helps!

+1 to what Sam said, all commits in a branch will show up in a PR. A PR pulls a branch into master, and that includes all commits made in that branch. If you want to avoid this, make a branch off of the branch.

GoogleyGabriel reviewed (approved):
Looking great! This is exactly how PRs should be -- keep up the hard work :)

GoogleyGabriel commented on portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java:
```diff
@@ -0,0 +1,18 @@
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/tester")
```
Change this to a more descriptive name, such as "fetch-hello-world-string".

sebagabs replied:
Just did so in a commit.

sebagabs committed:
Change @WebServlet name to be more descriptive; Update servlet name in message.js and index.html files

sebagabs committed:
Remove reference to message.js file in index.html file

sebagabs committed:
Add JavaScript function from message.js in script.js

sebagabs committed:
Delete message.js file and use script.js file instead

sebagabs commented:
I made some final commits for this pull request. Just changed some details @GoogleyGabriel pointed out, and modified the `script.js` file to include the function from `message.js`; got this I idea from Claudia's code. Also, I changed the `index.html` file so that it would not have a reference to `message.js` since now it does not exist.

sebagabs committed:
Fix use of quotation marks in const inside displayMessage() func in script.js

sebagabs committed:
Add comment to async function displayMessage()
